scalafix-core/src/main/scala/scalafix/rewrite/NoValInForComprehension.scala

```diff
@@ -0,0 +1,21 @@
+package scalafix
+package rewrite
+
+import scala.meta._
+import scala.meta.contrib._
+
+case object NoValInForComprehension extends Rewrite {
+
+  override def rewrite(ctx: RewriteCtx): Patch = {
+    ctx.tree.collect {
+      case f: Term.ForYield =>
```

Comment:
Wait, I don't think we need to match against `Term.For` in the first place. I think it's enough to match directly against `Enumerator.Val`